src/scripts/main.js

```diff
@@ -1,3 +1,340 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const headers = table.querySelectorAll('th');
+const tableBody = table.querySelector('tbody');
+
+headers.forEach((header) => {
+  header.dataset.sorted = 'DESC';
+});
+
+function sortColumn(index, isASC) {
+  let sortedRows = [];
+
+  if (isASC) {
+    // eslint-disable-next-line chai-friendly/no-unused-expressions
+    index === 4
+      ? sortedRows = Array.from(tableBody.rows).sort((a, b) =>
+        (salaryToNumber(a.cells[index].innerHTML)
+        - salaryToNumber(b.cells[index].innerHTML)))
+      : sortedRows = Array.from(tableBody.rows).sort((a, b) =>
+        (a.cells[index].innerHTML.localeCompare(b.cells[index].innerHTML)));
+  } else {
+    // eslint-disable-next-line chai-friendly/no-unused-expressions
+    index === 4
+      ? sortedRows = Array.from(tableBody.rows).sort((a, b) =>
+        (salaryToNumber(b.cells[index].innerHTML)
+        - salaryToNumber(a.cells[index].innerHTML)))
+      : sortedRows = Array.from(tableBody.rows).sort((a, b) =>
+        (b.cells[index].innerHTML.localeCompare(a.cells[index].innerHTML)));
```

Comment:
You use the same logic here. Maybe you need just to create additional variables for a and b and switch them if is not `ASC`. For example
```
Array.from(tableBody.rows).sort((a, b) => {
  const first = isASC ? a : b;
  const second = isASC ? b : a;

  return index === 4 ? ... : ...
}

It's just an example. Maybe you'll find a better solution